restaurants.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
could you please add more fields to restaurant? in real-world, the restaurant has an address, working hours... - as you described your entities in requirements document.
also, any entity should have its ID, so that we can index it and find it

Reply:
fixed